docs/shared/recipes/deployment/deploy-nuxt-to-vercel.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Deploying Nuxt applications to Vercel
+description: A detailed guide on how to deploy Nuxt applications from Nx workspaces to Vercel.
+---
+
+# Deploying Nuxt applications to Vercel
+
+## Configure your Nuxt application appropriately for Vercel deployment
+
+In your application's `nuxt.config.ts` file, ensure that the `nitro` property is configured to use the `vercel` preset, and that the `output.dir` property is set to the appropriate directory. The directory that Vercel expects to find the application's build output (as explained in the [Vercel documentation](https://vercel.com/docs/build-output-api/v3)) in is `.vercel/output` at the root of the repository/workspace. So, if your application is located at `apps/my-app` then the `output.dir` property should be set to `../../.vercel/output`.
+
+For example, if your Nx workspace is structured like this:
+
+```treeview
+apps/
+  my-app/
+    src/
+      main.ts
+    nuxt.config.ts
+nx.json
+package.json
+tsconfig.base.json
+```
+
+Then your `nitro` configuration should look like this:
+
+```ts {% fileName="apps/my-app/nuxt.config.ts" %}
+  nitro: {
+    preset: 'vercel',
+    output:{
+      dir: '../../.vercel/output'
+    }
+  },
+```
+
+For Vercel deployment, it does not really matter what the `buildDir` property in your `nuxt.config.ts` is set to.
+
+So, your entire `nuxt.config.ts` file may look something like this:
+
+```ts {% fileName="apps/my-app/nuxt.config.ts" %}
+import { nxViteTsPaths } from '@nx/vite/plugins/nx-tsconfig-paths.plugin';
+import { defineNuxtConfig } from 'nuxt/config';
+
+export default defineNuxtConfig({
+  workspaceDir: '../../',
+  srcDir: 'src',
+  devtools: { enabled: true },
+  buildDir: '../../dist/apps/my-app/.nuxt',
+  devServer: {
+    host: 'localhost',
+    port: 4200,
+  },
+  typescript: {
+    typeCheck: true,
+    tsConfig: {
+      extends: './tsconfig.app.json',
+    },
+  },
+  imports: {
+    autoImport: false,
+  },
+  css: ['~/assets/css/styles.css'],
+  vite: {
+    plugins: [nxViteTsPaths()],
+  },
+  nitro: {
+    preset: 'vercel',
+    output: {
+      dir: '../../.vercel/output',
+    },
+  },
+});
+```
+
+## Configure your Vercel project's settings appropriately
+
+### New Vercel project
+
+1. If you are "importing" your Nx workspace's repository for the first time, make sure you do _not_ choose a root directory as part of the repo selection process (therefore leaving it to be the root of the full repo/workspace)
+2. Ensure the Nuxt.js "Framework Preset" is selected
+3. Expand the "Build & Development Settings" and toggle the override switch for the build command. For example, for an application named `my-app` the value will look like this:
+
+```shell
+npx nx build my-app --prod
```

Comment:
I don't think we need `--prod` here, the build is defaulted to production config. (we also need to update the Next.js deploy docs separately)